scripts/examples/build-all-clusters-app.py

```diff
@@ -54,17 +55,28 @@ def main():
   e = IDFExecutor()
 
   if args.clear_config:
+    old_default_sdkconfig = None
+    clear_curr = args.clear_config != 'curr'
```

Comment:
I don't understand this line.  I'd expect `==` here, not `!=`.  If there is something deep here that I am missing about what "curr" means, it should at least be documented.

Reply:
This is just because the variable is strangely named - "clear_config" basically just re-runs menuconfig starting from the given default. If the default is "curr" (ie, re-run menuconfig, but starting from the current configuration), then we don't need to delete the current configuration and replace the default.

I tried to keep the name because I thought it would be less confusing, but given that this script has been broken for a while now, it would appear I'm likely either the only one or one of the only ones using this. If I renamed this something like menuconfig_with_default, would that work? it would change how the script gets invoked, but I'm not sure it'll actually affect anyone.

Reply:
I think fixing the naming would be helpful, yes!  I'm fine with a followup for that.